jerry-core/ecma/operations/ecma-exceptions.c

```diff
@@ -291,8 +292,15 @@ ecma_raise_standard_error_with_format (ecma_standard_error_t error_type, /**< er
       }
       else
       {
-        ecma_value_t str_val = ecma_op_to_string (arg_val);
-        arg_string_p = ecma_get_string_from_value (str_val);
+        arg_string_p = ecma_op_to_prop_name (arg_val);
```

Comment:
AFAIK this modification should be on it's own it is not related to the TypedArray.